ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-accounting.j2

```diff
@@ -23,6 +23,9 @@ aaa accounting exec default {{ aaa_accounting.exec.default.type }} group {{ aaa_
 {%     if aaa_accounting.system.default.type is arista.avd.defined and aaa_accounting.system.default.group is arista.avd.defined %}
 aaa accounting system default {{ aaa_accounting.system.default.type }} group {{ aaa_accounting.system.default.group }}
 {%     endif %}
+{%     if aaa_accounting.dot1x.default.type is arista.avd.defined and aaa_accounting.dot1x.default.group is arista.avd.defined %}
+aaa accounting dot1x default {{ aaa_accounting.dot1x.default.type }} group {{ aaa_accounting.dot1x.default.group }}
+{%     endif %}
```

Comment:
We don't do that on any of the others (we should), so maybe let this one go for now, and we can add `groups` to all of them and deprecate the singular group.

Reply:
I will try my best to implement the request from Tony.

Reply:
It currently turned out to be a little difficult to implement it right now supporting multiple groups.

Since the others currently also do not support multiple groups and from perspective of time for our current campus project, would it please be possible to start with the provided implementation?

If you agree,
should we stay with this data model
```
aaa_accounting:
  dot1x:
    default:
      type: start-stop
      group: RADIUS
      multicast: true
      logging: true
 ```
or should we already go for an expandable one:
```
  dot1x:
    default:
      type: start-stop
      group:
        - name: RADIUS
          multicast: true
          logging: true
 ```

Reply:
This EOS CLI is a bit confusing, and the documentation has a couple of typos which does not help :)
`logging` is a "method" similar to `group abc`, so it is _not_ a property of a group.
For some reason EOS does not have a guard against repeating the same method multiple times, so that is why `logging` shows up after every group.
I do not know how the ordering of `logging` vs. `group abc` might affect the behavior, so until we know that, I would suggest that we skip expanding for now and just implement what we need (one group with `group` key similar to the rest of the models).
A future data model should probably be referring to `methods` which can then either be a group (with or without multicast) or logging.
```
dot1x:
  default:
    type: start-stop
    methods:
      - method: < logging | group >
        group: < group_name >   <-- Only relevant for method group
        multicast: < bool >     <-- Only relevant for method group
```